.readthedocs.yml

```diff
@@ -0,0 +1,11 @@
+# Read the Docs configuration file
+# See https://docs.readthedocs.io/en/stable/config-file/v2.html for details
+
+version: 2
+build:
+  os: ubuntu-22.04
+  tools:
+    python: "3.8"
+  jobs:
+    post_create_environment:
+      - pip install .[docs]
```

Comment:
If you're just installing an extra, there is a special config section for that, see this example: https://github.com/angr/archinfo/blob/master/.readthedocs.yml#L14

Reply:
Ok..